soweego/linker/workflow.py

```diff
@@ -754,7 +753,7 @@ def _occupations_to_set(df):
     def to_set(itm):
         # if it is an empty array (from source), or an
         # empty string (from target)
-        if not itm:
+        if not itm or (isinstance(itm, float) and pd.isna(itm)):
```

Comment:
Just curious, why do you need these extra checks?

Reply:
When running the hyperparameter grid search there was one entry which had `NaN` as one of the values in the `occupations` column. I honestly didn't think much of it at the time so I just added this extra check.

However, it might be worthwhile opening an issue to investigate this further, maybe something weird happened during the importing process. What do you think?

Reply:
Sounds like extra sanity checks for malformed input due to a bad dump, or bad import, or bad query to the internal DB.
It would be great to log a WARNING when such unexpected input occurs.

Reply:
Good idea! Warning log added in 00ac61e